app/views/modals/create-secret.html

```diff
@@ -0,0 +1,10 @@
+<div class="create-secret-modal">
+  <div class="modal-header">
+    <h2>Create Secret</h2>
+  </div>
+  <div class="modal-body">
+    <ng-include ng-controller="CreateSecretController" src="'views/_create-secret.html'"></ng-include>
```

Comment:
Is this so the controller can be reused in a separate page? I'd rather make this a directive if it's meant to be a reusable component.